modules/learn/pages/clusters-and-availability/xdcr-overview.adoc

```diff
@@ -278,65 +278,87 @@ Detailed information is provided in xref:manage:manage-xdcr/monitor-xdcr-replica
 
 [#xdcr-compatibility]
 == XDCR Compatibility
+[.edition]#{enterprise}#
 
-The following table indicates XDCR compatibility between different versions of Couchbase Enterprise Server, used for source and target clusters.
+The following table indicates XDCR compatibility between different versions of Couchbase Server Enterprise edition, used for source and target clusters.
 
-[cols="6,3,3,3,3,3,3"]
+[cols="4,4,4,4"]
 |===
 | *Enterprise Server Version*
-| *7.6.x*
-| *7.2.x*
-| *7.1.4*, *7.1.3*, *7.1.2*, *7.1.1*
-| *7.1.0*
-| *7.0.x*
-| *6.6.x*
-
-| 7.6.x
-| ✓
-| ✓
-| ✓
-| ✓
-| ✓
-| ✓
+| *8.0.x*
+| *7.6.6 and later versions*
+| *7.6.5*, *7.6.4*, *7.6.3*, *7.6.2*, *7.6.1*, *7.6.0*, *7.2.x*, *7.1.x*, *7.0.x*
 
-| 7.2.x
-| ✓
-| ✓
-| ✓
-| ✓
-| ✓
+| 8.0.x
 | ✓
+| ✓*
+| ✓**
 
-| 7.1.4, 7.1.3, 7.1.2, 7.1.1
-| ✓
-| ✓
-| ✓
-| ✓
-| ✓
-| ✓
+No ECCV
 
-| 7.1.0
-| ✓
-| ✓
-| ✓
-| ✓
-| ✓
-| &#10060;
+| 7.6.6 and later
+| ✓*
+| ✓*
+| ✓**
 
-| 7.0.x
-| ✓
-| ✓
-| ✓
-| ✓
-| ✓
-| ✓
+No ECCV
 
-| 6.6.x
-| ✓
-| ✓
-| ✓
-| &#10060;
-| ✓
+| 7.6.5, 7.6.4, 7.6.3, 7.6.2, 7.6.1, 7.6.0, 7.2.x, 7.1.x, 7.0.x
+| ✓**
+
+No ECCV
+| ✓**
+
+No ECCV
 | ✓
 
 |===
+
+[NOTE]
+====
+XDCR Compatibility with vBucket Configuration (for both * and **)::
+
+Starting in Couchbase Server 8.0, Magma storage backend buckets use 128 vBuckets.
```

Comment:
This statement is not entirely correct.
Starting in Couchbase Server 8.0, Magma storage backend buckets can have either 128 vBuckets or 1024 vBuckets.